README.md

```diff
@@ -20,23 +20,23 @@ The current version of the toolkit requires:
 - Terraform 1.3 (or later)
 - Ansible 6.4 (or later)
 
-Take into account other Linux distributions might work but the ones listed
-here are the ones used to test the toolkit. Additionally, this documentation
-contains the steps necessary to install Terraform and Ansible.
+Take into account that other Linux distributions might work but the ones listed
+in this README file are the ones that have been used to test this particular toolkit. For your convenience, this documentation
```

Comment:
We usually limit the number of characters per line to 80. Would you mind to reduce the length of this line?